index.html

```diff
@@ -48,8 +50,238 @@ <h1 class="main-name">Erik Stoupignan</h1>
         <div class="prueba">
             <img src="/assets/image_geometry_2.png" alt="Decorative photo" class="background-img2">
         </div>
-
     </main>
+
+    <!-- Proyects section -->
```

Comment:
- [ ] Nice working on your `works` section, kindly make the container of your `works section` a `section` tag for better use of HTML semantics

Reply:
@danielufeli Good night Daniel! 
 I have already added the tag section to my "Work section". Thank you for your advice.

And... I have a question about the next point, I sent you a message for slack. Do yo have a chance for a Zoom call?